tests/ConfigurationTest.php

```diff
@@ -92,34 +90,19 @@ public function forSymmetricSignerShouldConfigureSignerAndBothKeys(): void
         self::assertSame($key, $config->verificationKey());
     }
 
-    /**
-     * @test
-     *
-     * @covers ::forUnsecuredSigner
-     * @covers ::signer
-     * @covers ::signingKey
-     * @covers ::verificationKey
-     */
-    public function forUnsecuredSignerShouldConfigureSignerAndBothKeys(): void
-    {
-        $key    = InMemory::empty();
-        $config = Configuration::forUnsecuredSigner();
-
-        self::assertInstanceOf(None::class, $config->signer());
-        self::assertEquals($key, $config->signingKey());
-        self::assertEquals($key, $config->verificationKey());
-    }
-
     /**
      * @test
      *
      * @covers ::builder
      *
-     * @uses \Lcobucci\JWT\Configuration::forUnsecuredSigner
+     * @uses \Lcobucci\JWT\Configuration::forSymmetricSigner
      */
     public function builderShouldCreateABuilderWithDefaultEncoderAndClaimFactory(): void
     {
-        $config  = Configuration::forUnsecuredSigner();
+        $config  = Configuration::forSymmetricSigner(
+            new KeyDumpSigner(),
```

Comment:
Is there a reason not to use the already existing algorithms? It would be nice to avoid creating these dummy implementations if they aren't strictly needed.

Reply:
1. To keep [this](https://github.com/lcobucci/jwt/pull/939/files#diff-0a93fdbee28c3e1d214d3b97133586005eb22068004ba68e47bf55dc386196dbR82) and [this](https://github.com/lcobucci/jwt/pull/939/files#diff-19b6967a3abe1a9b5eda4ed5af86c0e403722a6a92fbee56119641eecf8acccbR73) tests simple
2. To avoid polluting `tests/ConfigurationTest.php` with dozen of `@use`s